### ISSUE TITLE
Nvidia 435 with PRIME off load and Optimus Manager

### ISSUE THREAD

**Issue by michaldybczak** [Askannz/optimus-manager#121]
Since 435 nvidia drivers were released it begs the question, what about optimus-manager?

There is an interesting discussion and tests done here:

https://forum.manjaro.org/t/nvidia-435-17-driver-looks-exciting/98724

At first I thought: since optimus-manager does its own settings, I could switch to nvidia-435 series. However, so far there are some issues with those drivers:

- nvidia GPU isn't completely powered down (which isn't an issue with optimus-manager because we can always switch to pure intel mode when needed anyway)
- 435 series is not working well with Plasma DE
- new features are still very experimental and not complete
- 435 driver needs completely different configs and work in a very different way so the optimus-manager configs may not work well with it
- optimus-manager does some deep level systemd changes and I have no idea how it would play with 435 driver

All in all, I decided to stay on 430 series, which breaks CUDA but I don't use CUDA anyway, so for me, it's fine.

I suspect you also have no info or experience with those drivers, so I think the discussion showed above, especially various experiments on configurations may be of interest to you.

Manjaro made it easy to test it since it switched to driver series schema (testing branch), which gives a great opportunity to play with different drivers and setups.

I'm starting this topic to point you to the info you may be interested in, but also because at some point we need some guidance:

- What to do when using optimus-manager?
- How to upgrade on Manjaro to 435 series properly so it would work with optimus-manager (if that is possible or reasonable)?
- Is optimus-manager prepared for 435 series and the new features or maybe we should avoid for now?
etc.

So basically we need some clear readme section to know what to do, what are the perks or cons of switching to 435 series.

Also, considering that Arch users will switch to 435 drivers automatically at some point, will it break optimus-manager? What could they expect?

I find optimus-manager to be a wonderful piece of software. It may not work for everyone but if it works, it's just perfect so the new nvidia features won't replace it soon. I expect a long and painful process till those GPU off-loading features will be on pair with windows ones (if they ever be).

Anyway, enjoy the reading and hopefully, we see some of your output and information on optimus-manager with 435 dirver series (and probably all upcoming ones).

**Comment by Askannz** [comment]
See https://github.com/Askannz/optimus-manager/issues/114 for a discussion about integrating the new Prime offloading and power management features into optimus-manager.

As for 435.17 breaking optimus-manager, I honestly don't know what you are talking about. That driver did not introduce any incompatible change and it works fine for me in KDE Plasma.

**Comment by michaldybczak** [comment]
Ah, thanks. I just wanted to have the clear statements that optimus-manager will work with 435 series. As to Plasma, some reported having issues with this driver. Good to hear that it may not be the general problem.

I'll stay on 430 for some time and will observe how the situation resolves and switch to newer series later when I feel it's time, I'm happy with optimus-manager so at the moment I don' really need those new features, especially if they come with some cost of higher energy usage (reported 6% more, not much but still). 99% of my time I use intel as I game just occasionally so optimus-manager is just perfect.

If that is already being discussed elsewhere I'm closing this. Thanks for the info.
